### ISSUE TITLE
Have a Splunk 8.0 instance so hoping to get Pyden upgraded if enough interests

### ISSUE THREAD

**Issue by YiClareCao** [badarsebard/pyden-suite#11]
Cannot get Pyden work with our Splunk 8.0 instance and was told that it is not compatible. 
We love the idea using Pyden-manager/Pyden to get various Cpython built from source and all users can do PIP install in their now virtual environments.
Hope there is enough interests to get it upgraded!
Thanks
Clare

**Comment by christchow** [comment]
Would like to see Splunk 8 supported as well. This is a pretty useful app for me.

**Comment by rshahdelhi** [comment]
Splunk 8 support will be very helpful.

**Comment by badarsebard** [comment]
I'm gearing up to take this on soon. I've been creating a test framework to help aid me in development now that I'm planning to cover multiple versions. Especially since I will likely end up with concurrent major versions (1.x and 2.x) in order to remove the capability of compiling python2 once Splunk Inc. removes it from their product all together (guessing v8.1).

Test framework is complete. Just adding more actions to the driver. Expect to start in about a week. A previous PR from @owenBeresford indicates the app needs only small changes in order to support v8 of Splunk, at least minimally. However, this looks to still use python2 in v8 for the scripts in the manager app. My intention is to refactor these into python3 since there may be organizations that can no longer run python2 at all. Splunk allows a global forcing of all python execution to happen under the python 3 interpreter they bundle into v8. This is why it will take longer than just accepting the PR.

tl;dr
Stay tuned. Should only take a couple of weeks.

**Comment by owenBeresford** [comment]
Hi @badarsebard , Im glad you have come back to this project.

There is a minimal support version (patches to you);  OR my python3 branch.
My pytthon3 branch drops py2 support, adds the env-in-a-config file that you proposed, adds distinct exit codes for speed of problem solving.
I have been focussing on the features I was using.
I ran it against your tests, and the splunk inc test submission tests; it passes.

To above posters, this isn't in the splunk inc shop at present; but could be uploaded

**Comment by badarsebard** [comment]
Release 1.3.0 is out and certified on Splunkbase. This version supports 8.0.x under both the python2 and python3 bundled interpreters.
